dbcon/joblist/crossenginestep.cpp

```diff
@@ -546,23 +565,16 @@ void CrossEngineStep::execute()
                 applyMapping(fFeMapping3, rowFe3, &fRowDelivered);
 
                 addRow(rgDataDelivered);
+
+                if (rowFe3.usesStringTable() && fe3RGData.getStringTableMemUsage() > 50 * (1 << 20))
+                {
+                    fe3RGData.clearStringStore();
+                }
```

Comment:
We should move this threshold value (50 * (1 << 20)) into a common location (maybe in rowgroup.h) as a const int or a macro, so it can be easily updated later.

Use of UNLIKELY around the if predicate might be a suitable optimization?